From ddb9cce8a2ec87cdd9853bdca25c852c7aee8bdf Mon Sep 17 00:00:00 2001
From: Phillip Susi <psusi@ubuntu.com>
Date: Fri, 4 May 2018 10:35:42 -0400
Subject: [PATCH 84/92] tests: fix t6100-mdraid-partitions

The test was failing because it didn't wait for the md device
to appear after creating it.
---
 tests/t6100-mdraid-partitions.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/t6100-mdraid-partitions.sh b/tests/t6100-mdraid-partitions.sh
index dbb1686..b37cddf 100755
--- a/tests/t6100-mdraid-partitions.sh
+++ b/tests/t6100-mdraid-partitions.sh
@@ -47,6 +47,7 @@ cleanup_fn_() {
 
 # create mdraid on top of both partitions
 mdadm -C $md_dev --force -R -l1 -n2 "${scsi_dev}1" "${scsi_dev}2"
+wait_for_dev_to_appear_ ${md_dev} || { fail=1; cat /proc/partitions; }
 
 # create gpt and two partitions on the raid device
 parted -s $md_dev mklabel gpt \
-- 
2.17.1

